### DISCUSSION TITLE
2.0.4

### DISCUSSION THREAD

**Discussion by benhillis** [microsoft/WSL#10590] [General]
*  Resolve potential crash in wslservice.exe
*  Implement /usr/bin/wslinfo binary for querying various state, added `--networking-mode` argument
```
wslinfo usage:
    --networking-mode
        Display current networking mode.

    -n
        Do not print a newline.
```

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/WSL/releases/tag/2.0.4'>2.0.4</a>.</em>

**Comment by Ansuel** [comment]
Can confirm that this version fix the regression with 2.0.3 with explorer entry disappeared!

**Comment by bondarenkod** [comment]
Hello,

I encountered an issue while attempting to update to the latest version of WSL via the console - the update process was unsuccessful. Consequently, I downloaded and installed the update manually.

During the manual installation, an error message appeared, stating that it "couldn't create a shortcut for wsl... CLID" or something similar. Despite this error, it appears that the WSL version has been successfully updated on the system.

![image](https://github.com/microsoft/WSL/assets/3184414/17096de4-3043-4fae-9805-a69142322e60)
